zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -861,6 +862,11 @@ of servers -- that is, when deploying clusters of servers.
     connect and sync to a leader. Increased this value as needed, if
     the amount of data managed by ZooKeeper is large.
 
+* *connectToLearnerMasterLimit* :
+    (No Java system property)
+    Amount of time, in ticks (see [tickTime](#id_tickTime)), to allow followers to
+    connect to the learner master after leader election.
```

Comment:
We must tell about the default value

Reply:
Default value in the example config is 5. Also if the value is not specified or it's not greater than 0, then the `initLimit` value will be used instead.